### PR TITLE
Removed izip usage

### PR DESCRIPTION
**Pull request by richardjgowers**
Issue #632

### CODE REVIEW

**Comment by kain88-de** [comment]
You can also import map from six.

File: package/MDAnalysis/lib/qcprot.pyx [hunk 1]
```diff
@@ -181,7 +181,7 @@ def InnerProduct(np.ndarray[np.float64_t,ndim=1] A,
     A[0] = A[1] = A[2] = A[3] = A[4] = A[5] = A[6] = A[7] = A[8] = 0.0
 
     if (weight is not None):
-        for i in xrange(N):
```
Comment (jbarnoud) [inline package/MDAnalysis/lib/qcprot.pyx]: How is Cython dealing with `range` and `xrange`? Did Cython made the same change as Python regarding how `range` work?

**Comment by jbarnoud** [comment]
Except for my comment on Cython, it looks good to me.

File: package/MDAnalysis/lib/qcprot.pyx (hunk 1 above)
Comment (richardjgowers) [inline package/MDAnalysis/lib/qcprot.pyx]: Looking through the resultant C, it looks like both just become for loops in C

**Pull request merged**